functions-methods.js

```diff
@@ -33,4 +58,28 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    let valid = null;
+
+    if (valid === null || valid) {
+        valid = email.includes('@') ? email.indexOf('@') > 0 && email.indexOf('@') < email.length - 3 : false;
+    }
+
+    if (valid === null || valid) {
+        valid = !email.includes(',');
+    }
+
+    if (valid === null || valid) {
+        valid = email.lastIndexOf('.') < email.length - 1;
+    }
+
+    return valid;
```

Comment:
Het kost me even om te begrijpen wat hier gebeurt. Ik zou het zelf iets simpeler doen:
 ```suggestion
    let commaIsValid = email.includes('@') && email.indexOf('@') > 0 && email.indexOf('@') < email.length - 3;
    let containsComma = email.includes(',');
    let dotIsValid = email.includes('.') && email.lastIndexOf('.') < email.length - 1;
    
    return commaIsValid && !containsComma && dotIsValid;
```